### PR TITLE
Added Workflow to verify ambassaor.json schema

### PR DESCRIPTION
**Pull request by ramishj**
**GitHub Issue:** [#755 ]

**Summary**:
This pull request updates the GitHub Actions workflow to validate the `ambassadors.json` file on push and pull request events.

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**
No

### CODE REVIEW

**Review by Relequestual — APPROVED** [review]
This looks good, thanks for your work on this.

I have one condition on approving and one suggestion for the future.

Condition: When merging this, please select the "squash and merge" option.

Suggestion: Test on your local machine before pushing. Pushing should not be your test. You do not need to push every commit you make =]
You can squash commits locally by using git interactive rebasing. Happy to share some additional resources if that would be helpful.

You can test GitHub actions locally using [ACT](https://github.com/nektos/act).

File: programs/ambassadors/ambassadors-schema.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```
Comment (benjagm) [inline programs/ambassadors/ambassadors-schema.json:2]: Can we change the json schema version to use the current 2020-12?

File: .github/workflows/Validate Ambassadors JSON.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
```
Comment (benjagm) [inline .github/workflows/Validate Ambassadors JSON.yml:22]:
```suggestion
    - name: Set up Node 20
      uses: actions/setup-node@v4
      with:
        node-version: '20'
```

File: programs/ambassadors/ambassadors-schema.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  "type": "array",
```
Comment (benjagm) [inline programs/ambassadors/ambassadors-schema.json:3]: Can we add the description keyword to provide a basic description of each field?

File: programs/ambassadors/ambassadors-schema.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            "link": {
+              "type": "string",
+              "format": "uri"
+            }
+          },
+          "required": ["type", "title", "date", "link"]
+        }
+      }
+    },
+    "required": ["name", "img", "bio", "title", "github", "twitter", "linkedin", "company", "country", "contributions"]
```
Comment (benjagm) [inline programs/ambassadors/ambassadors-schema.json:75]: Let's make twitter and linkedin optional.

File: programs/ambassadors/ambassadors-schema.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        "type": "string"
+      },
+      "contributions": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string",
+              "enum": ["article", "talk", "video", "other"]
```
Comment (Relequestual) [inline programs/ambassadors/ambassadors-schema.json:42]: We have pending PRs which have more types than this.
@benjagm did you want to expand this list (and/or the list in the document)?

**Review by benjagm — APPROVED** [review]
Awesome!! Very good job with this!!

I just left some comments with small suggestions.

**Review by benjagm — CHANGES_REQUESTED** [review]
I left some comments and suggestions.

Amazing job with this!!

File: programs/ambassadors/ambassadors-schema.json (hunk 5 above)
Comment (benjagm) [reply programs/ambassadors/ambassadors-schema.json:42]: Lets add:
- book
- paper
- initiative
- project
- working group

**Comment by ramishj** [comment]
> I left some comments and suggestions.
> 
> Amazing job with this!!

I have made the required changes you suggested . 
Thanks for the suggestions and would love to keep contribute .
Thanks ....

**Comment by ramishj** [comment]
> This looks good, thanks for your work on this.
> 
> I have one condition on approving and one suggestion for the future.
> 
> Condition: When merging this, please select the "squash and merge" option.
> 
> Suggestion: Test on your local machine before pushing. Pushing should not be your test. You do not need to push every commit you make =] You can squash commits locally by using git interactive rebasing. Happy to share some additional resources if that would be helpful.
> 
> You can test GitHub actions locally using [ACT](https://github.com/nektos/act).

Thanks for you suggestion @Relequestual would keep in my mind in future  while testing for workflows .

File: programs/ambassadors/ambassadors-schema.json (hunk 1 above)
Comment (ramishj) [reply programs/ambassadors/ambassadors-schema.json]: I tried updating it but got this error while testing should i chnage the script and manually check for each entry in schema?

Comment (ramishj) [reply programs/ambassadors/ambassadors-schema.json]: @benjagm Sir , could please advice what should we do...

**Review by benjagm — APPROVED** [review]
Nice job! Thanks!

File: .github/workflows/Validate Ambassadors JSON.yml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        with:
+          node-version: '20'
+
+      - name: Install Dependencies
+        run: npm install ajv ajv-formats
+
+      - name: Validate ambassadors.json
+        run: |
+          node -e "
+          const Ajv = require('ajv');
```
Comment (benjagm) [inline .github/workflows/Validate Ambassadors JSON.yml:30]: To run the validation against a 2020-12 schema the dependency to use is: `ajv/dist/2020`

Right now the action is failing:
https://github.com/json-schema-org/community/actions/runs/9631882401

**Review by benjagm — CHANGES_REQUESTED** [review]
The workflow is failing after updating to the current draft `2020-12`. I think the dependency to use is other:
https://ajv.js.org/json-schema.html#draft-2020-12

Logs:
https://github.com/json-schema-org/community/actions/runs/9631882401

File: programs/ambassadors/ambassadors-schema.json (hunk 1 above)
Comment (JeelRajodiya) [reply programs/ambassadors/ambassadors-schema.json]: @ramishj, The image you uploaded is not visible, could you try uploading again  please?
image

**Review by benjagm — APPROVED** [review]
Great job @ramishj ! This is a cool feature very much needed.

**Pull request merged**